tests/cases/standalone/common/flow/flow_basic.result

```diff
@@ -45,6 +45,7 @@ ADMIN FLUSH_FLOW('test_numbers_basic');
 |  FLOW_FLUSHED  |
 +----------------------------------------+
 
+-- SQLNESS SLEEP 1s
```

Comment:
So many sleep 😱

Reply:
> So many sleep 😱

tmp fix, will create a separate PR to remove this